docs/source/model-registry.rst

```diff
@@ -171,10 +179,69 @@ as shown below to create a new version of the model.
         run_id="d16076a3ec534311817565e6527539c0",
     )
 
+.. _using-registered-model-aliases:
```

Comment:
When I built the webpage locally, I kept on getting this error
```
sphinx.errors.SphinxWarning: source/model-registry.rst:184:duplicate label using-registered-model-aliases, other instance in /Users/jerry.liang/mlflow/docs/source/model-registry.rst
```
Even though this is the only place with this label. I'm adding because there are many places in the product that reference it and we should make sure they still work. Any ideas on why this error is happening? @harupy @BenWilson2